Isodose/MRML/vtkMRMLIsodoseNode.cxx

```diff
@@ -88,10 +88,8 @@ void vtkMRMLIsodoseNode::ReadXMLAttributes(const char** atts)
   vtkMRMLReadXMLBeginMacro(atts);
   vtkMRMLReadXMLBooleanMacro(ShowIsodoseLines, ShowIsodoseLines);
   vtkMRMLReadXMLBooleanMacro(ShowIsodoseSurfaces, ShowIsodoseSurfaces);
-  vtkMRMLReadXMLBooleanMacro(ShowScalarBar, ShowScalarBar);
-  vtkMRMLReadXMLBooleanMacro(ShowScalarBar2D, ShowScalarBar2D);
   vtkMRMLReadXMLBooleanMacro(ShowDoseVolumesOnly, ShowDoseVolumesOnly);
-  vtkMRMLReadXMLIntMacro(DoseUnits, DoseUnits);
+  vtkMRMLReadXMLEnumMacro(DoseUnits, DoseUnits);

```

Comment:
Wouldn't changing the Read/Write DoseUnits macro from Int to Enum break compatibility with existing saved scenes?

Reply:
I will check that.

Reply:
Backward compatibility (new versions of SlicerRT can read both old and new scenes) can be maintained by implementing the `Get##propertyName##FromString(xmlReadAttValue)` method so that it recognizes old integer values.

This breaks future compatibility (scenes that are created in new version of SlicerRT will not be fully readable in older versions), but this may be acceptable, especially since extensions auto-update is now available in Slicer5 (making it easier for users to keep SlicerRT up-to-date).